src/ensembl/src/services/analytics-service.ts

```diff
@@ -49,6 +50,9 @@ class AnalyticsTracking {
       transport: 'xhr',
       value: ga.value
     });
+
+    typeof ga.species === 'string' && this.setSpeciesDimension('');
+    typeof ga.feature === 'string' && this.setFeatureDimension('');
```

Comment:
1) Do you remember why we write `typeof something === 'string' && ...` rather than simply `something && ...`. It would make sense if we were considering empty string to be a meaningful input, which would fail the `something && ...` check; but we aren't passing empty strings to this method.

2) Does unsetting a field mean setting it to empty string or to `undefined`? The docs are very confusing 😞